### PR TITLE
completed admin waiver edit page

### PR DESCRIPTION
**Pull request by jdaniel0x50**
Completed vue page for waiver edits (same page will edit camper waiver and volunteer waiver based on routing)

### CODE REVIEW

**Review by spyeadon — APPROVED** [review]
Hey looks good to me!

File: src/components/admin_waiver_edit.vue [hunk 1]
```diff
@@ -439,8 +439,26 @@
...
+          'title': this.waiver.title,
+          'headers': this.waiver.header,
+          'items': this.waiver.items,
+          'footer': this.waiver.footer
+        }
+        localforage.getItem('X_TOKEN')
+        .then(session => {
+          axios.put('/api/v1/admin/waiver/' + this.type + '/update', {
+            headers: { 'x-token': session },
+            dataObj: dataObj
```
Comment (zachgarwood) [inline src/components/admin_waiver_edit.vue]: I don't see `dataObj` in the [request configuration options](https://github.com/axios/axios#request-config), did you mean `data`?

**Review by zachgarwood — CHANGES_REQUESTED** [review]

File: src/components/admin_waiver_edit.vue (hunk 1 above)
Comment (jdaniel0x50) [reply src/components/admin_waiver_edit.vue]: sounds good - I changed the key to 'data' here and on the backend route - it makes more sense to use this consistently

**Review by spyeadon — APPROVED** [review]
Makes sense

**Review by zachgarwood — APPROVED** [review]

**Comment by zachgarwood** [comment]
@JennaDalgety don't forget to add your review!

**Comment by JennaDalgety** [comment]
Ah!  Apologies.  Thought I had responded to this.  Looked great in the video chat last week and Zach's edit should be the icing on the cake!

**Pull request merged**